qa-tests-backend/src/main/groovy/services/ApiTokenService.groovy

```diff
@@ -1,9 +1,11 @@
 package services
 
+import groovy.util.logging.Slf4j
```

Comment:
Could this be done one the BaseService.groovy to apply to all Services or does the decorator have to happen on a class by class basis?

Reply:
Nevermind. I see your comment on the other PR.